### ISSUE TITLE
Run supernode creation and visualization on locally created graph

### ISSUE THREAD

**Issue by guhanrv** [decoderesearch/circuit-tracer#20]
I have created a local `.pt` file and am looking to do visualizations like those at the bottom of https://github.com/safety-research/circuit-tracer/blob/main/demos/circuit_tracing_tutorial.ipynb without a neuronpedia URL. Is there a way to create supernodes and `create_graph_visualization` for an arbitrary `.pt` file?

An example snippet would be incredibly helpful. Thank you!

**Comment by mntss** [comment]
Yes, this is possible. You'll need to convert your `.pt` file to graph JSON files first, then serve them locally:

```bash
circuit-tracer start-server --graph_file_dir /path/to/graph_files
```

Alternatively, you can do everything in one command:

```bash
circuit-tracer attribute \
  --prompt "your original prompt" \
  --transcoder_set gemma \
  --slug your-graph-name \
  --graph_file_dir ./graph_files \
  --server
```

This will create the graph files and start the server where you can label nodes, create supernodes, and use the local URL with `create_graph_visualization`.

**Comment by mntss** [comment]
1. We don't have methods to do automated supernode grouping currently. This is a great research direction and we would be very interested in hearing about any approaches you develop or seeing contributions if you decide to work on it!

2. The token predictions come from the LLM we're analyzing. We pick the top-k most likely next tokens (or until we reach `desired_logit_prob` cumulative probability) using `compute_salient_logits`. These are the model's own predictions for what token should come next.
  For your "High glucose" example, " high" genuinely is the model's most probable next token. If you want to analyze "diabetes" specifically, you have a couple options:

    * Modify your prompt to make "diabetes" a more likely next token
    * Modify the attribution code to target "diabetes" specifically instead of using top logits
    * Use arbitrary directions like probe vectors trained on intermediate activations

    Right now circuit tracing works for fixed prompts and we only support output logits at last position. Multi-token attribution isn't supported, though the underlying algorithm could be extended to attribute to any direction at any position in the model activations. These extensions are possible with small changes to the attribution code but aren't supported by the frontend currently.
